common/compositor/factory.cpp

```diff
@@ -17,46 +17,106 @@
 #include "factory.h"
 #include "platformdefines.h"
 
-#ifdef USE_GL
 #include "glrenderer.h"
 #include "glsurface.h"
 #include "nativeglresource.h"
-#elif USE_VK
```

Comment:
We might still want a way to disable vulkan on platforms which dont support vulkan. On our side we can enable USE_VK as default and give a board option to disable it if needed/configure option to disable it on Linux ? be->is_supported() would always return false for vulkan in case support is disabled